Makefile

```diff
@@ -54,27 +54,27 @@ test: node_modules/.uptodate
 
 .PHONY: test-py3
 test-py3: node_modules/.uptodate
-	tox -e py36 -- tests/h/
+	tox -e py36-tests
```

Comment:
`tests/h` is given in the `tox.ini` file, doesn't need to be repeated in the `tox` command. You _can_ specify the path / test(s) to run if you want to run a subset though: `tox -e py36-tests tests/h/accounts/` (note the `--` is unnecessary).